<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scheduleListWidget</class>
 <widget class="QWidget" name="scheduleListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="taskTime">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>00시 - 00시</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="taskName">
      <property name="text">
       <string>task</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="showBtn">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>보기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delBtn">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
